packages/testing/README.md

```diff
@@ -98,3 +98,46 @@ Steps are a convenient way to craft async tests. A step has a timeout and a desc
 | [Predicate](https://wixplosives.github.io/core3-utils/testing.predicate)     | <p>A predicate function</p><p>Any return value other than \*\*false\*\* or throwing is considered as satisfying the predicate</p> |
 | [Stub](https://wixplosives.github.io/core3-utils/testing.stub)               | A generated stub                                                                                                                  |
 | [Timeout](https://wixplosives.github.io/core3-utils/testing.timeout)         | Sets step timeout                                                                                                                 |
+
+## Chai Retry Plugin
+
+Plugin that allows to re-run function passed to `expect` until chained assertion will be passed or timeout exceeded or retries limit reached.
```

Comment:
Mention that assertions will be run agains value returned from the function

Reply:
@yurii-ve fixed